web/client/localConfig.json

```diff
@@ -100,6 +100,29 @@
           }
         }
     }, {
+        "name": "Tutorial",
+        "cfg": {
+          "rawSteps": [
+            {
+              "translation": "intro",
+              "selector": "intro"
+            },
+            {
+              "translation": "lorem",
```

Comment:
In the default configuration we should already have some minimal introduction, we can customize and extend later, not a lorem ipsum